packages/functional-tests/pages/relier.ts

```diff
@@ -34,6 +34,13 @@ export class RelierPage extends BaseLayout {
     ]);
   }
 
+  clickSignIn() {
+    return Promise.all([
+      this.page.locator('button.sign-in-button.signin').click(),
+      this.page.waitForNavigation({ waitUntil: 'load' }),
+    ]);
+  }
+
```

Comment:
Unrelated to anything (and definitely not blocking), but I noticed the page has no copyright header.

Reply:
Alot of pages don't have it. I will create a separate PR to fix that once done with a chunk of tests.